core/src/main/java/io/grpc/Metadata.java

```diff
@@ -642,9 +645,9 @@ private static String validateName(String n) {
       return n;
     }
 
-    private Key(String name) {
+    private Key(String name, boolean pseudo) {
```

Comment:
It seems the user of that header is now no longer using it.  Can this PR proceed?